pagopa/io-app
# chore(IT Wallet): [SIW-636] Update provider list screen with new DS

## Short description
This PR updates `ItwDiscoveryProviderInfoScreen` with new DS


https://github.com/pagopa/io-app/assets/49342144/d4852a56-7e79-4344-bd37-882ab5488b7c



## How to test
Enable experimental DS System. Start activation flow. The `ItwDiscoveryProviderInfoScreen` should now use new DS components.
